migration_challenge_keras_image/util/nb.py

```diff
@@ -30,10 +30,11 @@ def upload_data(out):
             f"Uploading files from {local_path} to {s3_uri} in the background...\n"
         )
         t0 = time.time()
-        proc = subprocess.Popen(
-            ["aws", "s3", "sync", "--quiet", "--delete", local_path, s3_uri],
+        print(f"Uploading files from {local_path} to {s3_uri} in the background...\n")
```

Comment:
This either duplicates the `append_stdout()` line left in above, or one of the two is broken?

Reply:
Removed the duplication in 784e8457d0fd008d8d074575975a939c2aae1e18 but didn't check if `print()` *should* be the way it's solved...